plugins/woocommerce-beta-tester/src/features/index.js

```diff
@@ -23,6 +23,12 @@ function Features() {
 
 	const sortedFeatureNames = Object.keys( features ).sort();
 
+	const afterToggleCallback = {
+		'new-product-management-experience': () => {
+			location.reload();
```

Comment:
Just a style suggestion... `window.location.reload()`. It makes it immediately clear what `location` is.